lib/daru/accessors/dataframe_by_vector.rb

```diff
@@ -0,0 +1,21 @@
+module Daru
```

Comment:
no need for this file. It was removed for a reason. The reason being that default access on a DF is column-wise and specifying `vector` everytime is slow and not very useful